alert.css

```diff
@@ -0,0 +1,37 @@
+@import url(components.css);
+.content{
+    margin: 10px;
+    padding: 10px;
+    font-size: 1.5rem;
+    font-weight: 700;
+    text-align: center;
+}
+.content-background
+{
+    height: fit-content;
+    width: 900px;
+    border-radius: 15px;
+    box-shadow:5px 10px 8px #888888;
```

Comment:
use variable if you are re-using this color someplace else aswell

Reply:
done with changes